spi/src/main/java/org/mapstruct/spi/AccessorNamingStrategy.java

```diff
@@ -0,0 +1,102 @@
+/**
+ *  Copyright 2012-2015 Gunnar Morling (http://www.gunnarmorling.de/)
+ *  and/or other contributors as indicated by the @authors tag. See the
+ *  copyright.txt file in the distribution for a full listing of all
+ *  contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.mapstruct.spi;
+
+/**
+ * A service provider interface for the mapping between method names and properties.
+ *
+ * @author Christian Schuster
+ */
+public interface AccessorNamingStrategy {
+
+    /**
+     * Determine if a method name defines a getter with a non-boolean return type.
+     *
+     * @param methodName The method name.
+     * @return <code>true</code> if the method name can be a non-boolean getter, <code>false</code> otherwise.
```

Comment:
Gunnar will probably point that out as well: we prefer using ```{@code true}``` instead of ```<code>true</code>``` because with ```{@code ...}```, you can include HTML-special characters without having to use the entities. E.g. ```{@code x < y}``` would have to be written as ```<code>x &lt; y</code>``` otherwise.